test2.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
```

Comment:
__[development][rubocop]__
* Missing frozen string literal comment.
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->